rust/noosphere/src/ffi/context.rs

```diff
@@ -181,16 +188,23 @@ pub fn ns_sphere_content_read(
                     _ => return Ok(None),
                 };
 
-                if Peer::None != slashlink.peer {
-                    return Err(anyhow!("Peer in slashlink not yet supported"));
-                }
-
                 let slug = match slashlink.slug {
                     Some(slug) => slug,
                     None => return Err(anyhow!("No slug specified in slashlink!")),
                 };
 
-                let cursor = sphere.inner();
+                let cursor = match slashlink.peer {
```

Comment:
Function doc should be updated to reflect slashlink support of non-local spheres/petnames